synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -447,6 +457,75 @@ dragging the handle to the left bottom part of your 2D space."));
 	
 }
 
+void
+Dialog_Setup::create_shortcuts_page(Dialog_Template::PageInfo pi)
+{
+	int row = 1;
+
+	Gtk::Label *label = manage(new Gtk::Label(_("To change a shortcut, double-click it and then type the new keys.\n"
```

Comment:
Maybe remove new line after first message? As for me it looks better because there no free space on the right side, and message is more compact.

![Screenshot_149](https://user-images.githubusercontent.com/5604544/119248672-f4308b80-bbbc-11eb-8c00-743cc6f32ade.png)

![Screenshot_150](https://user-images.githubusercontent.com/5604544/119248673-f8f53f80-bbbc-11eb-8e40-2e7bb99f017f.png)

Reply:
Done